CONTRIBUTING.md

```diff
@@ -97,7 +116,7 @@ running live tests.
 To run live tests against a service after deploying live resources:
 
 ```
-mvn -f sdk/keyvault/pom.service.xml -Dmaven.wagon.http.pool=false --batch-mode --fail-at-end --settings eng/settings.xml test
+mvn -f sdk/keyvault/pom.xml -Dmaven.wagon.http.pool=false --batch-mode --fail-at-end --settings eng/settings.xml test
```

Comment:
Why don't we use -pl -am for live tests here? Or mention dependencies need to cache in local before run this?

Reply:
Let's say you are working on EventHubs ... do you really want to spin up all the storage resources for your EventHubs test (because of checkpoint store?). Nothing here is stopping you do it but I'm trying to make it simple for folks.

Of course if you are making changes across storage and event hubs then you'll probably want to do just that. Although in that case you'd probably run Maven against the root and specify both sets of modules on the -pl switch.